FoldingTabBar/View/FoldingTabBar/YALFoldingTabBar.h

```diff
@@ -36,16 +38,12 @@ typedef NS_ENUM(NSUInteger, YALTabBarState) {
 
 @interface YALFoldingTabBar : UIView
 
-- (instancetype)initWithFrame:(CGRect)frame state:(YALTabBarState)state;
-
-@property (nonatomic, weak) IBOutlet id<YALTabBarViewDataSource> dataSource;
-@property (nonatomic, weak) IBOutlet id<YALTabBarViewDelegate> delegate;
+- (instancetype)initWithController:(YALFoldingTabBarController *)controller;
 
 @property (nonatomic, assign, readonly) YALTabBarState state;
 @property (nonatomic, assign) NSUInteger selectedTabBarItemIndex;
 
 @property (nonatomic, copy) UIColor *tabBarColor;
-@property (nonatomic, copy) UIColor *dotColor;
```

Comment:
Please put back this property or provide some another way how to customize the colour of doc for selected VC. Thanks.